### PR TITLE
docs(getting-started): generate SSH key ed25519 even on Windows

### PR DESCRIPTION
**Pull request by EtienneM**

### CODE REVIEW

File: src/_posts/platform/getting-started/2000-01-01-setup-ssh-linux.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Setup SSH on Linux
-modified_at: 2021-10-25 00:00:00
+modified_at: 2022-01-06 00:00:00
```
Comment (curzolapierre) [inline src/_posts/platform/getting-started/2000-01-01-setup-ssh-linux.md:3]: Is it wanted to update the date here?

File: src/_posts/platform/getting-started/2000-01-01-setup-ssh-macos.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Setup SSH on Mac OS
-modified_at: 2015-04-04 00:00:00
+modified_at: 2022-01-06 00:00:00
```
Comment (curzolapierre) [inline src/_posts/platform/getting-started/2000-01-01-setup-ssh-macos.md:3]: Same as above

**Review by curzolapierre — APPROVED** [review]
Just a question, otherwise LGTM

File: src/_posts/platform/getting-started/2000-01-01-setup-ssh-linux.md (hunk 1 above)
Comment (EtienneM) [reply src/_posts/platform/getting-started/2000-01-01-setup-ssh-linux.md:3]: Yep it was forgotten during last update. I checked when this file was last modified and updated it.

**Pull request merged**